.github/workflows/publish.yml

```diff
@@ -12,11 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: 'master'
       - uses: actions/setup-node@v1
       - run: |
           git config user.name "${{ secrets.USER_NAME }}"
           git config user.email "${{ secrets.USER_EMAIL }}"
-          git clone -b master "https://${{ secrets.USER_PAT }}@github.com/act-rules/act-rules-web"
+          git clone -b master "https://82fc6b9d6b970ad0ace26f922dcb90dba83477bf@github.com/act-rules/act-rules-web"
```

Comment:
Is that a secret that is now included in the URL?

Reply:
@Jym what happens when a secret is pushed by mistake is that GitHub revokes the token.
So I regenerated a new one after. All good.